Sources/SwiftSyntax/Syntax.swift

```diff
@@ -228,6 +228,10 @@ extension SyntaxProtocol {
     return _syntaxNode.data.raw
   }
 
+  public func detach() -> Self {
```

Comment:
Could you add a doc comment to explain what this method does?

Reply:
Sure